utils/hct/hctdb.py

```diff
@@ -3245,8 +3260,62 @@ def add_attr_arg(title_name, scope, args, doc):
         add_attr_arg("Unroll", "l", "Unroll the loop until it stops executing or a max count", [{"name":"Count", "type":"int"}])
         self.attributes = attributes
 
+class InstructionSignature:
+    def __init__(self, fn_attr, ret_type, name, args):
+        self.fn_attr = fn_attr
+        self.ret_type = ret_type
+        self.name = name
+        self.args = args
+    def __str__(self):
+        str_args = ",".join(self.args)
+        return "[{}] {} {}({})".format(self.fn_attr, self.ret_type, self.name, str_args)
+
+def execfile(filepath, globals=None, locals=None):
```

Comment:
As of the latest commit, this is no longer used. I think I'd prefer if it is though.